spec/factories/issue.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+	factory :issue do
+		sequence(:title) { |n| "Issue title #{n}"  } 
```

Comment:
Unnecessary spacing detected.<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.<br>Trailing whitespace detected.